db/seeds.rb

```diff
@@ -36,8 +36,8 @@
 # end
 
 
-#Retrieving the list of all user ids
-p user_ids_list = User.all.pluck(:id)
+#Retrieving the list of all user ids, excluded Mark from the list as we will be using his ID for the demo and we do not want him to own costumes or to have reservations before the demo.
+p user_ids_list = User.all.where.not(name:"Jean-Francois").pluck(:id)
```

Comment:
Mark or Jean-Francois?

Reply:
Comment should say Jean-Francois, will update!

Reply:
Hey @andking , update is pushed